### PR TITLE
add contributing to PyMC

### PR DESCRIPTION
**Pull request by SangamSwadiK**
## Description: What does this implement/fix? Explain your changes.
 
Adds contributing to PyMC

## Reference: Issue or Pull Request (PR)
- [x] Towards #87 

## Reminders

- [x] Did you review the [CONTRIBUTING GUIDE](https://github.com/data-umbrella/data-umbrella-website/blob/main/CONTRIBUTING.md)?
- [x] Did you reference an issue or another pull request?

#### Any other comments?
None

### CODE REVIEW

**Comment by reshamas** [comment]
@SangamSwadiK Thanks for this PR.
a) I think for filters, we should use:
- contributing (~~video~~)
- using-pymc (~~pymc-resource~~)

b) For ArviZ thumbnail, we can use this one as a start (may need resizing):
https://www.canva.com/design/DAFTXBr6mxg/9QnMuzbCHObGl049DcdJpg/edit?utm_content=DAFTXBr6mxg&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

c) For "Contributing to PyMC Documentation", let's start with this thumbnail:
https://www.canva.com/design/DAE6rZJAdA0/GB-izEd0N7fTbbCWZ6gIrA/edit?utm_content=DAE6rZJAdA0&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

c) In the navigation bar, I would move "Contributing to PyMC" above the "core python" one.

d) Can we update this title to "PyMC" and not "PyMC3" (remove ref to version 3):
Intuitive Bayesian modeling with PyMC3 

e) The current links in the 3rd row point to the YouTube video.  I think we can point them to:
- Intuitive Bayesian Modeling with PyMC: https://pymc-data-umbrella.xyz/en/latest/about/intuitive_modeling/index.html
- etc.

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      sub_title: "PyMC-Resource"
+      sm_img: "/assets/images/data-umbrella/pymc/intro-to-probabilistic-programming.png"
+      big_img: "/assets/images/data-umbrella/pymc/intro-to-probabilistic-programming.png"
+      url: "https://youtu.be/Qu6-_AnRCs8"
+      resource: "PyMC-Resource"
+      resource_url: ""
+      filters:
+        - PyMC-Resource
+
+    - title: "Intuitive Bayesian modeling with PyMC3"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:29]:
```suggestion
    - title: "Intuitive Bayesian Modeling with PyMC"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      sub_title: "PyMC-Resource"
+      sm_img: "/assets/images/data-umbrella/pymc/intuitive-bayesian-modelling.png"
+      big_img: "/assets/images/data-umbrella/pymc/intuitive-bayesian-modelling.png"
+      url: "https://youtu.be/6dc7JgR8eI0"
+      resource: "PyMC-Resource"
+      resource_url: "https://youtu.be/6dc7JgR8eI0"
+      filters:
+        - PyMC-Resource
+
+    - title: "An example Pull Request(PR) to PyMC"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:39]:
```suggestion
    - title: "An Example Pull Request (PR) to PyMC"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      sub_title: "Video"
+      sm_img: "/assets/images/data-umbrella/pymc/contributing-to-pymc.png"
+      big_img: "/assets/images/data-umbrella/pymc/contributing-to-pymc.png"
+      url: "https://youtu.be/Iq0dY5hU4D4"
+      resource: "Video"
+      resource_url: "https://youtu.be/Iq0dY5hU4D4"
+      filters:
+        - video
+
+    - title: "Contributing to PyMC documentation"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:59]:
```suggestion
    - title: "Contributing to PyMC Documentation"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+      sub_title: "PyMC-Resource"
+      sm_img: "/assets/images/data-umbrella/pymc/arviz.png"
+      big_img: "/assets/images/data-umbrella/pymc/arviz.png"
+      url: "https://youtu.be/457ZTes4xOI"
+      resource: "PyMC-Resource"
+      resource_url: "https://youtu.be/457ZTes4xOI"
+      filters:
+        - PyMC-Resource
+
+    - title: "Intro to Numpy array operations"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:79]:
```suggestion
    - title: "Intro to Numpy Array Operations"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+  portfolios:
+
+    - title: "Intro to Probabilistic Programming with PyMC"
+      sub_title: "PyMC-Resource"
+      sm_img: "/assets/images/data-umbrella/pymc/intro-to-probabilistic-programming.png"
+      big_img: "/assets/images/data-umbrella/pymc/intro-to-probabilistic-programming.png"
+      url: "https://youtu.be/Qu6-_AnRCs8"
+      resource: "PyMC-Resource"
+      resource_url: ""
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:25]:
```suggestion
      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/probabilistic_programming_with_pymc/index.html"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+      filters:
+        - PyMC-Resource
+
+    - title: "Intuitive Bayesian modeling with PyMC3"
+      sub_title: "PyMC-Resource"
+      sm_img: "/assets/images/data-umbrella/pymc/intuitive-bayesian-modelling.png"
+      big_img: "/assets/images/data-umbrella/pymc/intuitive-bayesian-modelling.png"
+      url: "https://youtu.be/6dc7JgR8eI0"
+      resource: "PyMC-Resource"
+      resource_url: "https://youtu.be/6dc7JgR8eI0"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:35]:
```suggestion
      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/intuitive_modeling/index.html"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+      filters:
+        - video
+
+    - title: "Contributing to PyMC"
+      sub_title: "Video"
+      sm_img: "/assets/images/data-umbrella/pymc/contributing-to-pymc.png"
+      big_img: "/assets/images/data-umbrella/pymc/contributing-to-pymc.png"
+      url: "https://youtu.be/Iq0dY5hU4D4"
+      resource: "Video"
+      resource_url: "https://youtu.be/Iq0dY5hU4D4"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:55]:
```suggestion
      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_pymc/index.html"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+      filters:
+        - PyMC-Resource
+
+    - title: "Intro to Numpy array operations"
+      sub_title: "PyMC-Resource"
+      sm_img: "/assets/images/data-umbrella/pymc/numpy-operations.png"
+      big_img: "/assets/images/data-umbrella/pymc/numpy-operations.png"
+      url: "https://youtu.be/oud3Jd1FJ7c"
+      resource: "PyMC-Resource"
+      resource_url: "https://youtu.be/oud3Jd1FJ7c"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:85]:
```suggestion
      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/intro_to_array_operations/index.html"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+
+    - title: "Contributing to ArviZ and Open Source"
+      sub_title: "using-pymc"
+      sm_img: "/assets/images/data-umbrella/pymc/arviz-contrib.png"
+      big_img: "/assets/images/data-umbrella/pymc/arviz-contrib.png"
+      url: "https://youtu.be/457ZTes4xOI"
+      resource: "using-pymc"
+      resource_url: "https://youtu.be/457ZTes4xOI"
+      filters:
+        - using-pymc
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:77]:
```suggestion
        - contributing
        - using-pymc
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_pymc/index.html"
+      filters:
+        - contributing 
+
+    - title: "Contributing to PyMC Documentation"
+      sub_title: "contributing "
+      sm_img: "/assets/images/data-umbrella/pymc/pymc-docs-contrib.png"
+      big_img: "/assets/images/data-umbrella/pymc/pymc-docs-contrib.png"
+      url: "https://youtu.be/fzpmLWQNj4A"
+      resource: "contributing "
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:64]:
```suggestion
      resource: "contributing"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+      sm_img: "/assets/images/data-umbrella/pymc/contributing-to-pymc.png"
+      big_img: "/assets/images/data-umbrella/pymc/contributing-to-pymc.png"
+      url: "https://youtu.be/Iq0dY5hU4D4"
+      resource: "contributing "
+      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_pymc/index.html"
+      filters:
+        - contributing 
+
+    - title: "Contributing to PyMC Documentation"
+      sub_title: "contributing "
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:60]:
```suggestion
      sub_title: "contributing"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/example_pr/index.html"
+      filters:
+        - contributing 
+
+    - title: "Contributing to PyMC"
+      sub_title: "contributing "
+      sm_img: "/assets/images/data-umbrella/pymc/contributing-to-pymc.png"
+      big_img: "/assets/images/data-umbrella/pymc/contributing-to-pymc.png"
+      url: "https://youtu.be/Iq0dY5hU4D4"
+      resource: "contributing "
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:54]:
```suggestion
      resource: "contributing"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+      sm_img: "/assets/images/data-umbrella/pymc/pymc-pr.png"
+      big_img: "/assets/images/data-umbrella/pymc/pymc-pr.png"
+      url: "https://youtu.be/NbmdFJsnuuo"
+      resource: "contributing "
+      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/example_pr/index.html"
+      filters:
+        - contributing 
+
+    - title: "Contributing to PyMC"
+      sub_title: "contributing "
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:50]:
```suggestion
      sub_title: "contributing"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 14]
```diff
@@ -0,0 +1,141 @@
...
+      sm_img: "/assets/images/data-umbrella/pymc/intuitive-bayesian-modelling.png"
+      big_img: "/assets/images/data-umbrella/pymc/intuitive-bayesian-modelling.png"
+      url: "https://youtu.be/6dc7JgR8eI0"
+      resource: "using-pymc"
+      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/intuitive_modeling/index.html"
+      filters:
+        - using-pymc
+
+    - title: "An Example Pull Request (PR) to PyMC"
+      sub_title: "contributing "
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:40]:
```suggestion
      sub_title: "contributing"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 15]
```diff
@@ -0,0 +1,141 @@
...
+      filter: "using-pymc"
+
+  portfolios:
+
+    - title: "Intro to Probabilistic Programming with PyMC"
+      sub_title: "using-pymc"
+      sm_img: "/assets/images/data-umbrella/pymc/intro-to-probabilistic-programming.png"
+      big_img: "/assets/images/data-umbrella/pymc/intro-to-probabilistic-programming.png"
+      url: "https://youtu.be/Qu6-_AnRCs8"
+      resource: "using-pymc"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:24]:
```suggestion
      resource: "Outline and resources"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 16]
```diff
@@ -0,0 +1,141 @@
...
+      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/probabilistic_programming_with_pymc/index.html"
+      filters:
+        - using-pymc
+
+    - title: "Intuitive Bayesian Modeling with PyMC"
+      sub_title: "using-pymc"
+      sm_img: "/assets/images/data-umbrella/pymc/intuitive-bayesian-modelling.png"
+      big_img: "/assets/images/data-umbrella/pymc/intuitive-bayesian-modelling.png"
+      url: "https://youtu.be/6dc7JgR8eI0"
+      resource: "using-pymc"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:34]:
```suggestion
      resource: "Outline and resources"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 17]
```diff
@@ -0,0 +1,141 @@
...
+      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/intuitive_modeling/index.html"
+      filters:
+        - using-pymc
+
+    - title: "An Example Pull Request (PR) to PyMC"
+      sub_title: "contributing "
+      sm_img: "/assets/images/data-umbrella/pymc/pymc-pr.png"
+      big_img: "/assets/images/data-umbrella/pymc/pymc-pr.png"
+      url: "https://youtu.be/NbmdFJsnuuo"
+      resource: "contributing "
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:44]:
```suggestion
      resource: "Outline and resources"
```

**Review by reshamas — CHANGES_REQUESTED** [review]

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 18]
```diff
@@ -0,0 +1,142 @@
...
+      filters:
+        - contributing
+        - using-pymc
+
+    - title: "Intro to Numpy Array Operations"
+      sub_title: "using-pymc"
+      sm_img: "/assets/images/data-umbrella/pymc/numpy-operations.png"
+      big_img: "/assets/images/data-umbrella/pymc/numpy-operations.png"
+      url: "https://youtu.be/oud3Jd1FJ7c"
+      resource: "using-pymc"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:85]:
```suggestion
      resource: "Outline and resources"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 19]
```diff
@@ -0,0 +1,142 @@
...
+      filters:
+        - contributing 
+
+    - title: "Contributing to ArviZ and Open Source"
+      sub_title: "using-pymc"
+      sm_img: "/assets/images/data-umbrella/pymc/arviz-contrib.png"
+      big_img: "/assets/images/data-umbrella/pymc/arviz-contrib.png"
+      url: "https://youtu.be/457ZTes4xOI"
+      resource: "using-pymc"
+      resource_url: "https://youtu.be/457ZTes4xOI"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:75]:
```suggestion
      resource_url: "https://youtu.be/457ZTes4xOI"
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 20]
```diff
@@ -0,0 +1,142 @@
...
+---
+
+
+<div class="isotope-filter text-center">
+  <br>
+  <br>
+<!-- <h3 class="section-title mb-40 text-center> -->
+ <h4>
+  We host <a href="https://pymc-data-umbrella.xyz/en/latest/2023-03_sprint/index.html">PyMC sprints and working sessions</a> regularly.  
+</h4>
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:98]:
```suggestion
  We have extensive resources on <a href="https://pymc-data-umbrella.xyz/en/latest/2023-03_sprint/index.html">Contributing to PyMC</a>.  
</h4>
```

File: snowlake-v1.2/open-source/contributing-to-pymc.html (hunk 19 above)
[the remark this answers is not in the document]
Comment (OriolAbril) [reply snowlake-v1.2/open-source/contributing-to-pymc.html:75]: I did not but could do it if you prefer linking there than linking to the video directly

File: snowlake-v1.2/open-source/contributing-to-pymc.html [hunk 21]
```diff
@@ -0,0 +1,142 @@
...
+      resource_url: "https://pymc-data-umbrella.xyz/en/latest/about/contributing_to_documentation/index.html"
+      filters:
+        - contributing 
+
+    - title: "Contributing to ArviZ and Open Source"
+      sub_title: "using-pymc"
+      sm_img: "/assets/images/data-umbrella/pymc/arviz-contrib.png"
+      big_img: "/assets/images/data-umbrella/pymc/arviz-contrib.png"
+      url: "https://youtu.be/457ZTes4xOI"
+      resource: "using-pymc"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/contributing-to-pymc.html:74]:
```suggestion
      resource: "video"
```

**Comment by reshamas** [comment]
Thank you @SangamSwadiK.
Nice to have the PyMC resources on one page! I think people will find it useful.

**Pull request merged**